README.md

```diff
@@ -216,6 +216,15 @@ The getting started walkthrough illustrates the interactive login experience, wh
 - **[Visit our sample repos](docs/examples/samples.md) for full projects with different use cases**
 - [Configuring API Management](docs/examples/apim.md) that sits in front of function app
 
+### Logging Verbosity
+
+You can set the logging verbosity with the `--verbose` flag (`-v` for short). If the `--verbose` flag is set with no value, logging will be as verbose as possible (debug mode). You can also provide a value with the flag to set the verbosity to a specific level:
```

Comment:
Is the default mode debug here? It seems that in the logging service, the default is 'info'

Reply:
Default is `info`. `debug` is set if the `--verbose` flag is passed with no argument, or if there is an argument, it sets it to that value